core/src/models/Property.ts

```diff
@@ -372,10 +372,14 @@ export class Property extends LoggedModel<Property> {
     const count = await Property.count({
       where: {
         id: { [Op.ne]: instance.id },
-        key: instance.key,
+        key: Sequelize.where(
+          Sequelize.fn("LOWER", Sequelize.col("key")),
+          instance.key.toLowerCase()
+        ),
```

Comment:
Yep, that works!

Are there any other `ensureUnique*` methods we should update in a similar way?
A quick glance shows:
* `App.ensureUniqueName`
* `Destination.ensureUniqueName`
* `Group.ensureUniqueName`
* `Schedule.ensureUniqueName`
* `Source.ensureUniqueName`

Reply:
Good question. We should expand these into a story in our backlog.